src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        ColorSupplier ballColor = new ColorSupplier();
```

Comment:
move random and colorSupplier to the class field, like:
![image](https://user-images.githubusercontent.com/12864128/210410304-d4f5eb71-d071-46d6-933c-f47dcb6d9e6e.png)

Reply:
done